model.py

```diff
@@ -3991,7 +3991,7 @@ def for_unchecked_subjects(cls, _db):
             LicensePool.identifier).join(
                 Identifier.classifications).join(
                     Classification.subject)
-        return qu.filter(Subject.checked==False).distinct()
+        return qu.filter(Subject.checked==False).order_by(Subject.id)
```

Comment:
This improves both reliability and performance. Previously the goal was to check every work only once. Now the goal is to check all the works associated with a subject at once. If the script crashes, it's less likely that a bunch of subjects will have been marked as checked without all the associated works being reclassified.

Getting rid of the `distinct` improves performance of the query quite a bit, but it means that sometimes a work will be reclassified more than once (because it was classified under two unchecked subjects). That's okay because reclassification is fast.